Tools/scripts/mavlink_parse.py

```diff
@@ -7,7 +7,7 @@
 from dataclasses import dataclass, astuple
 
 from pymavlink.dialects.v20 import (
-    common, icarous, cubepilot, uAvionix, ardupilotmega
+    common, icarous, cubepilot, uAvionix, ardupilotmega, development
```

Comment:
should be just be 'all'? either that or needs minimal
does this end up with lots of duplicates via the recursion of ardupilot -> common -> minimal?

Reply:
> should be just be 'all'?

It's nice to break them out into dialects, so that the message definitions can be linked to directly within the MAVLink documentation.

> either that or needs minimal

The messages defined in `minimal` [are duplicated in the `common` doc](https://mavlink.io/en/messages/common.html#HEARTBEAT), so I didn't think it was necessary to break it out further (because the links already work fine). 

Happy to make that change if you think it's preferable to link to the minimal docs instead of the common docs for those couple of messages. Thoughts?

> does this end up with lots of duplicates via the recursion of ardupilot -> common -> minimal?

Nope - the dialects are checked in order (see `determine_dialect`), so it just stops when it gets to the first one that matches. The `MAVLinkDetector` class maps relevant messages to their dialects (and corresponding documentation links) as needed, so the only message duplicates are if the firmware handles the same message in different ways that are checked for (e.g. `SYSTEM_TIME` can be incoming, (automatically) outgoing, and also requestable, so appears in all three tables).